index.html

```diff
@@ -2244,11 +2244,12 @@ <h3>Terms of Use</h3>
 Terms of use can be utilized by an <a>issuer</a> or a <a>holder</a> to
 communicate the terms under which a <a>verifiable credential</a> or
 <a>verifiable presentation</a> was issued. The <a>issuer</a> places their terms
-of use inside the <a>credential</a>. The <a>holder</a> places their terms of use
-inside a <a>presentation</a>.  The value of this property tells the
-<a>verifier</a> what actions it must perform (an <em>obligation</em>), must not
-perform (a <em>prohibition</em>), or may perform (a <em>permission</em>) if it
-is to accept the <a>verifiable credential</a> or <a>verifiable presentation</a>.
+of use inside the <a>verifiable credential</a>. The <a>holder</a> places their
+terms of use inside a <a>verifiable presentation</a>. The value of this property
```

Comment:
This language that describes actions such as "must", "must not",  and "may" sounds very normative.

Reply:
Given the sentence is about the actions verifiers are _obliged_, _prohibited_, or _permitted_ to perform, I think this is OK in this context.

Reply:
@ken-ebert is correct -- it *sounds* normative, and that makes reviewers very cranky. We have to avoid using the normative trigger words "must", "must not", "should", "should not", etc. Can you apply some alternative wording, please? Otherwise, we're going to get objections (even though the language was never intended to be normative).

Reply:
using "required to" instead of "must", "not allowed to" instead of "must not", and "allowed to" instead of "may"